### ISSUE TITLE
Erfc Fails to be monotonically decreasing

### ISSUE THREAD

**Issue by 0xJepsen** [primitivefinance/solstat#15]
Erfc is decreasing [FAIL]
```
  x1 0
  y1 1000000030000000450
  x2 57896044618658097711785492504343953926634992332820282019727636134502933591130
  y2 1000000000001000160870564596263
```
x increases, but y doesn't decrease

**Comment by Autoparallel** [comment]
Ran
`yarn cli --cdf 01
`yarn cli --cdf 57896044618658097711785492504343953926634992332820282019727636134502933591130`
and got 
`0.5000000150000002`
`1`
respectively.

**Comment by clemlak** [comment]
Not closing it now because approximations are causing this issue.

**Comment by Alexangelj** [comment]
Fixed in #29 via [9d92706](https://github.com/primitivefinance/solstat/pull/29/commits/9d92706fe73dbef46e0b1884f634f78f1a8b792b) with the additional checks added to `erfc`.
